docs/architecture/incremental.md

```diff
@@ -1,20 +1,20 @@
-# Incremental
+# Incremental Sync
```

Comment:
you should wait for charles PR before merging this one

Reply:
merged.